torch/nn/modules/conv.py

```diff
@@ -10,7 +11,8 @@
 class _ConvNd(Module):
 
     def __init__(self, in_channels, out_channels, kernel_size, stride,
-                 padding, dilation, transposed, output_padding, groups, bias):
+                 padding, dilation, transposed, output_padding, groups, bias,
+                 initializer):
```

Comment:
we could provide a default value here of `initializer = {}`, then in the constructor we can just do `self.initializer = initializer` and in `reset_parameters` we can take advantage of the default response of the`get` method on dicts. i.e. `reset_parameters` would be something like:

```python
self.initializer.get("weight", self._initializer)(self.weight)
if bias is not None:
    self.initializer.get("bias", self._initializer(self.bias)
```